api/controllers/PromotionController.js

```diff
@@ -35,7 +37,22 @@ var PromotionController = {
             })
 
         })
-    }
+    },
+
+    getRestaurantById: function(req, res) {
+        var id = req.swagger.params.id.value;
+        var token = req.headers.authorization;
+
+        superagent
+        .get('https://api.yelp.com/v3/businesses/'+ id)
+        .set('Authorization', token)
```

Comment:
this `token` coming from API user is the auth token issued by our api server, not the token by yelp

Reply:
Actually it is the api-key I got from Yelp.
![capture](https://user-images.githubusercontent.com/12376493/34087401-d0374020-e370-11e7-93b8-d636f0de908a.PNG)

Reply:
Sorry I don't understand, this `token` is from request header right? And how do you pass Yelp api key to the request header?

Reply:
Oh, wait a sec.
I should not put this API-KEY in the request header.
Cliend side should not know it.
It should only be known by server, correct?

Reply:
ya that's confusing me

Reply:
you can hardcode it for now, I have a PR coming soon regarding the config